api/nameguard/web_api.py

```diff
@@ -367,11 +469,19 @@ async def inspect_labelhash_post(request: InspectLabelhashRequest) -> NameGuardR
     },
 )
 async def secure_primary_name_get(address: str, network_name: NetworkName) -> SecurePrimaryNameResult:
+    """
+    ## Reverse lookup of Ethereum `address` to primary name.
+
+    This endpoint checks primary name using Provider API.
```

Comment:
Other endpoints refer to ENS Subgraph. It is also internal comment and indirectly says that you need to configure provider URI.

Reply:
@lightwalker-eth It is used in public docs but it is also internal comment - it can't be separated. Comments for other endpoints refer to ENS Subgraph, so if we remove here information about using provider then there we should remove information about using ENS subgraph?